library/Mockery/MockInterface.php

```diff
@@ -81,6 +81,21 @@ public function shouldDeferMissing();
     public function makePartial();
 
     /**
+     * Disable mocking non-existent methods
+     *
+     * @return Mock
+     */
+    public function disallowMockingNonExistentMethods();
```

Comment:
Adding methods to an interface is a breaking change btw. We'd need a 0.10 release for this.